bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/action/LGWebOSActions.java

```diff
@@ -237,6 +237,12 @@ public void decreaseChannel() {
         getConnectedSocket().ifPresent(control -> control.channelDown(createResponseListener()));
     }
 
+    @RuleAction(label = "@text/actionSendRCButtonLabel", description = "@text/actionSendRCButtonDesc")
+    public void sendRCButton(
```

Comment:
please also add a corresponding static method below 
// delegation methods for "legacy" rule support

Reply:
OK done. Please review ... again :)